third_party/lasread/lidario.go

```diff
@@ -479,8 +480,23 @@ func (las *LasFile) readHeader() error {
 	offset += 8
 	las.Header.MinZ = math.Float64frombits(binary.LittleEndian.Uint64(b[offset : offset+8]))
 	offset += 8
-	if las.Header.VersionMajor == 1 && las.Header.VersionMinor == 3 {
+	if las.Header.VersionMajor == 1 && las.Header.VersionMinor >= 3 {
 		las.Header.WaveformDataStart = binary.LittleEndian.Uint64(b[offset : offset+8])
+		offset += 8
+		las.Header.StartOfFirstExtendedVariableLengthRecord = binary.LittleEndian.Uint64(b[offset : offset+8])
+		offset += 8
+		las.Header.NumberOfExtendedVariableLengthRecords = binary.LittleEndian.Uint32(b[offset : offset+4])
+		offset += 4
+		las.Header.NumberOfPointRecords = binary.LittleEndian.Uint64(b[offset : offset+8])
+		offset += 8
+		for i := 0; i < 15; i++ {
+			las.Header.NumberOfPointsByReturn[i] = binary.LittleEndian.Uint64(b[offset : offset+8])
+			offset += 8
+		}
+	}
+
+	if las.Header.NumberPoints < 1 {
+		las.Header.NumberPoints = int(las.Header.NumberOfPointRecords)
```

Comment:
A bit unclear on why this is here.

Reply:
I can verify it again. Sorry, I'm not familiar with GO.